docs/api/en/lights/shadows/DirectionalLightShadow.html

```diff
@@ -82,6 +82,11 @@ <h3>[property:Camera camera]</h3>
 			clipping plane at 0.5 and the [page:OrthographicCamera.far far] clipping plane at 500.
 		</p>
 
+		<h3>[property:Boolean isDirectionalLightShadow]</h3>
+		<p>
+			Read-only flag to check whether a given object is of type [name].
```

Comment:
Instead,

    Read-only flag to check if a given object is of type [name].

Reply:
great.

Reply:
@ycw are you going to update this PR with @WestLangley suggestion or will you create a new PR?

Reply:
create new PR, including updates for all other tasks :D